modules/platforms/dotnet/Apache.Ignite.Core/Impl/Cache/Event/CacheEntryCreateEvent.cs

```diff
@@ -76,5 +77,16 @@ public CacheEntryEventType EventType
         {
             get { return CacheEntryEventType.Created; }
         }
+
+        /// <summary>
+        /// Returns a <see cref="string" /> that represents this instance.
+        /// </summary>
+        /// <returns>A <see cref="string" /> that represents this instance.</returns>
+        public override string ToString()
+        {
+            return string.Format(CultureInfo.CurrentCulture, 
+                "{0} type Cache Event [Key = {1}, HasValue = {2}, Value = {3}, HasOldValue = {4}, OldValue = {5}]", 
```

Comment:
Let's follow the established format (e.g. see `AffinityKey`): `CacheEntryCreateEvent [Key={}, ...]` - note that only the type name goes outside of square brackets, and properties go inside, without spaces around `=` (here and below).